src/component/Banner.tsx

```diff
@@ -345,7 +417,7 @@ class Banner extends Component<Props, State> {
                                         }}
                                         aria-expanded={showInfo}
                                         aria-controls={INFO_LIST_ID}
-                                        tabIndex={1}
+                                        tabIndex={2}
```

Comment:
I see the `tabIndex` has been increased by 1 because of the introduction of the `buttonAsLink` element which now occupies index 1 for users' in the `variant`, this makes sense. However if a user is in the `control` or not in the test at all this `buttonAsLink` element won't exist, so `tabIndex` 1 will be missing so I'd be I'd interested to know how the tabbing behaves.

Reply:
since they're in order, they behave the same